dist/datasource.ts

```diff
@@ -17,7 +18,7 @@ export default class RocksetDatasource {
     this.backendsrv = backendSrv;
     this.headers = {'Content-Type': 'application/json'};
     this.headers['Authorization'] = `ApiKey ${instanceSettings.jsonData['apiKey']}`;
-    this.url = 'https://api.rs2.usw2.rockset.com';
+    this.url = instanceSettings.jsonData['apiServer'] || 'https://api.rs2.usw2.rockset.com';
```

Comment:
We are in the process of making api.use1a1.rockset.com the default except when we absolutely have to stay with rs2. See https://github.com/rockset/rs/pull/12642/files

Probably doesn't matter which one in this case so I would do use1a1